reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -239,3 +274,106 @@ private fun List<ResolvedLicense>.merge(): ResolvedLicense {
         locations = flatMapTo(mutableSetOf()) { it.locations }
     )
 }
+
+/**
+ * Return an [OrtResult] with all license and copyright findings associated with [targetProjects] removed from all other
+ * projects not in [targetProjects].
```

Comment:
Please emphasize that this affects projects which have a target project in a subdirectory, this would help understanding what the function does.